frontend/src/components/Conduct.vue

```diff
@@ -5,8 +5,8 @@
         <p class="max-w-3xl mt-2 mx-auto text-xl text-gray-500 p-4">
             At Python WA, we want to create a community that is friendly and inviting for newbies and professionals alike. 
             This is why we have adopted the 
-            <a class="text-blue-700" href="https://www.djangoproject.com/conduct/" target="_blank">
-                Django Code of Conduct
+            <a class="text-blue-700" href="https://policies.python.org/python.org/code-of-conduct/#:~:text=and%20growing%20community.-,Our%20Community,patches%2C%20problems%2C%20or%20otherwise" target="_blank">
```

Comment:
Is there a reason why this url was used and not just `https://policies.python.org/python.org/code-of-conduct/` ? 
I'm not sure I'm seeing the value in that text being highlighted?

Reply:
The fragment helps to highlight the first paragraph when the site is visited. If the highlight is not needed I can remove it.

Reply:
yes, please do!